### PR TITLE
Network settings ref

### LINKED ISSUE #231: Document "Network settings"
**Path to page with content (_do not_ include domain name):**
`/en/os/1.14.x/api/settings/`

**What content is incorrect?**
The settings reference lacks the "[Network Settings](https://github.com/bottlerocket-os/bottlerocket#network-settings)"

**What should the content say or communicate?**
There is an [ongoing effort](https://github.com/bottlerocket-os/bottlerocket-project-website/issues/203) to move the settings reference currently located on the [Bottlerocket README](https://github.com/bottlerocket-os/bottlerocket) to Bottlerocket.dev

This can be accomplished using the [Settings Documentation DSL](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/METADOC-SETTING-DOCS-DSL.md). Effectively, the settings written out in the README's markdown needs to be converted to this DSL format. You can see an example of how this looks in the [ECS settings reference TOML file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/data/settings/1.14.x/ecs.toml) as well as [the actual ECS content markdown file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/content/en/os/1.14.x/api/settings/ecs/_index.markdown).

The file should be located at `/data/settings/1.14.x/network.toml` and `/content/en/os/1.14.x/api/settings/network/_index.markdown`.

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes #231 

**Description of changes:**

Adds setting reference for (`settings.network.*`)

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: data/settings/1.15.x/network.toml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+[[docs.ref.no-proxy]]
+description = """
+A list of hosts that Bottlerocket will excluded from proxying.
+
+The no-proxy list automatically includes entries for `localhost`.
+
+On Kubernetes variants (`*-k8s-*`) the no-proxy lists includes the Kubernetes API server endpoint as well as other commonly used Kubernetes DNS suffixes.
+"""
+accepted_values = [
+    "List of host names and IP addresses (with optional port numbers)"
```
Comment (stmcginnis) [inline data/settings/1.15.x/network.toml:86]: I'm not sure if port numbers are used in `no-proxy`. I thought it was just at the whole-host level, but maybe someone more familiar can confirm.

Comment (stockholmux) [reply data/settings/1.15.x/network.toml:86]: @stmcginnis that makes some sense. 

@webern Do you know? It looks like you [committed some code around this](https://github.com/bottlerocket-os/bottlerocket/commit/73d3778e543f16254cca06e1293f6cf28454da97) back in the day.

Comment (webern) [reply data/settings/1.15.x/network.toml:86]: I'm not entirely sure. I searched around a bit but proxying traffic isn't something I ever do. It's probably best to leave out the reference to ports. Our proxying should work like most proxying does.

**Review by stmcginnis — APPROVED** [review]
Looks good to me!

**Pull request merged**